File: Libreria.Core/Entities/IBook.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Libreria.Core.Entities
{
    public interface IBook
    {
        int BookId { get; }
        string Title { get; }
        DateTime Date { get; }
        decimal Spend { get; }
        decimal SugestedPrice { get; }
        string Autor { get; }
        Editorial Editorial { get; }
    }
}

File: Libreria.view/src/app/book-module/entities/Ibook.ts
import {Ieditorial} from "./Ieditorial";

export interface Ibook{
    bookId : number;
    title:string;
    date:Date;
    spend:number;
    sugestedPrice:number;
    autor:string;
    editorialId:number;
    editorial:Ieditorial|null;
}

